src/DeribitSolution/Commands/AsyncSpectreCommand.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Logging;
+using Spectre.Console.Cli;
+using Spectre.Console;
+
+namespace ConsoleApp.Commands;
+
+internal class AsyncSpectreCommand : AsyncCommand<AsyncSpectreCommand.Settings>
+{
+    public override async Task<int> ExecuteAsync(CommandContext context, Settings settings)
+    {
+        var table = new Table().Centered();
+        await AnsiConsole.Live(table)
+            .StartAsync(async ctx =>
+            {
+                var num = 1;
+                table.AddColumn("Number");
+                table.AddColumn("Name");
+                table.AddColumn("Last Name");
+                ctx.Refresh();
+                await Task.Delay(1000);
```

Comment:
Can we delete this delay?

Reply:
Yes, it is a hardcoded sample, it will be update with real structure